audformat/core/database.py

```diff
@@ -514,11 +522,51 @@ def join_field(
                 elif value2:
                     self.__dict__[field] = value2
 
+        def update_scheme(
```

Comment:
One thing that is not so nice about implementing it as a method of `Scheme` is that it will change both - `self` and `other`, which might be a bit unusual. Maybe it makes more sense to have it as a utility function, e.g. `utils.join_schemes(...)`?

Reply:
Good point.

In my opinion it shouldn't update `other`, so maybe we have to work with a copy instead, if this is possible?

Reply:
But since changing the scheme labels also changes the table content this means we have to copy the whole database, don't think that is a good idea.

So maybe it's better that the user has to explicitly change the schemes in both database before calling `update()`. Something like:

```python
labels = audformat.utils.join-labels([db.schemes['scheme'].labels, [db_new.schemes['scheme'].labels])
db.schemes['scheme'].replace_labels(labels)
db_new.schemes['scheme'].replace_labels(labels)
db.update(db_new)
```

The downside is that it's of course a bit lengthy, but at least there are now hidden side-effects the user is not aware of.

Reply:
It's a good point.

I updated the pull request and propose now only to introduce a helper function.